tox.ini

```diff
@@ -34,3 +34,7 @@ commands = pre-commit run --all-files
 description = Run the development server with auto-reload for code changes.
 usedevelop = true
 commands = uvicorn s3proxy.main:app --reload
+pass_env =
+    LSST_RESOURCES_S3_PROFILE_*
```

Comment:
Where does this get used, and what does it do?

Reply:
`lsst.resources` uses that to work out which S3 profile is meant to be used for auth.